server/etc/pulp/server.conf

```diff
@@ -295,6 +295,10 @@
 # certfile: The absolute path to the PEM encoded certificate used for authentication to the message
 #     bus. The default value is '/etc/pki/pulp/qpid/client.crt'.
 #
+# broker_login_method: Select the login method used to connect to the broker. This should be left
+#     unset (use the broker default) except in special cases (e.g. do SSL client certificate
+#     authentication. For RabbitMQ with SSL client certificates, this needs to be 'EXTERNAL'.
```

Comment:
The last sentence should probably be moved to a new section on RabbitMQ with SSL and Client Cert Verification at the bottom of this doc:  http://pulp.readthedocs.org/en/latest/user-guide/broker-settings.html#rabbitmq-with-ssl